### ISSUE TITLE
Crash on joining

### ISSUE THREAD

**Issue by wrefgtzweve** [Vurv78/Autorun-rs#23]
```
<---> Autorun-rs <--->
Type [help] for the list of commands
Failed to allocate console. 5
<---> Autorun-rs <--->
Type [help] for the list of commands
Failed to open Autorun module. [Failed to set sender.]
Loaded into menu state.
Joining Server with IP XXXXXXXXXX!
warning: Unknown nb_ctl request:  4
warning: Unknown nb_ctl request:  4
warning: Unknown nb_ctl request:  4
warning: Unknown nb_ctl request:  4
warning: Unknown nb_ctl request:  4
warning: Unknown nb_ctl request:  4
thread '<unnamed>' panicked at 'byte index 3 is not a char boundary; it is inside '?' (bytes 2..3) of `note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: Rust panics must be rethrown
```

It crashed right after I loaded in, I was able to see a motd & the game in the background.

Using https://github.com/Vurv78/Autorun-rs/releases/tag/v0.6.0

**Comment by Vurv78** [comment]
Problem with the double loading trying to fill an already occupied cell again, most likely fixed in v0.7.0

**Comment by Vurv78** [comment]
Since this has already presumably been fixed in 0.7.0 and that's been preceded there's no point in keeping this open
